src/plugins/intel_cpu/src/graph_optimizer.cpp

```diff
@@ -1473,6 +1473,26 @@ void GraphOptimizer::FuseConvolutionAndDWConvolution(Graph &graph) {
     }
 }
 
+#if defined(OPENVINO_ARCH_ARM64)
+static bool is_post_op_supported(const std::shared_ptr<Node> &node) {
```

Comment:
Lets use existing `isUnarySupportedAsPostOp` for the same purposes. Maybe list inside isUnarySupportedAsPostOp should be adjusted

Reply:
fixed, list is updated